clients/drcachesim/common/options.cpp

```diff
@@ -374,6 +374,12 @@ droption_t<std::string> op_tracer_ops(
     "(For internal use: sweeps up tracer options)",
     "This is an internal option that sweeps up other options to pass to the tracer.");
 
+droption_t<int>
+    op_only_thread(DROPTION_SCOPE_FRONTEND, "only_thread", 0,
```

Comment:
Is there an easy way for the user to find out which tids are available?

Reply:
One specific use case is a thread id printed out by the invariant checker tool.  Other sources include from the file names of the thread files, from the basic_counts tool, or an unfiltered view run.  I think there are lots of sources and use cases.